src/components/Accordion/component.tsx

```diff
@@ -0,0 +1,74 @@
+import { useState } from "react";
+import { Minus, ChevronDown } from 'lucide-react';
+
+export type AccordionDataItem = {
+	title: string;
+	content: string | AccordionDataItem[];
+};
+
+export type AccordianDataItems = AccordionDataItem[];
+
+export interface AccordionItemProps {
+	title: string;
+	children: React.ReactNode;
+	expanded?: boolean;
+	headerStyle?: {};
+}
+
+const AccordionItem = ({ title, children, expanded, headerStyle }: AccordionItemProps) => {
+	const [isOpen, setIsOpen] = useState(expanded);
+	const [isActive, setIsActive] = useState(expanded);
```

Comment:
afaict these should be the same variable OR one should be derived from the other?